packages/shared/lib/core/wallet/utils/formatTokenAmountDefault.ts

```diff
@@ -5,6 +5,6 @@ export function formatTokenAmountDefault(amount: number, tokenMetadata: ITokenMe
     return (
         formatNumber(amount / 10 ** tokenMetadata?.decimals, 0, tokenMetadata?.decimals, 0, true) +
         ' ' +
-        tokenMetadata.unit
+        tokenMetadata?.unit
```

Comment:
If there is a case where this is `undefined` then it will append that to the string (e.g. 100 undefined). Is that desired?

Reply:
Not desired, I've fixed it :smile: